lib/tasks/routes.rake

```diff
@@ -1,6 +1,7 @@
 unless defined?(Rails)
   desc 'display all routes for Grape'
   task routes: :environment do
+    require './config/application.rb'
```

Comment:
Small fix here to re-enable the listing of Grape routes for a Gris app.

I think we may want to deprecate/remove this altogether (as this will break in versioned APIs) but at the very least this will load the default ApplicationEndpoint for new gris apps.